docs/development.rst

```diff
@@ -64,6 +64,15 @@ will override the defaults.
 Server Development
 ------------------
 
+All commits to the core python code must work in all versions of python 2.7 and above.
+Python code in plugins should also work in 2.7 and 3.x, but some plugins may depend
```

Comment:
I think we should claim 3.x. Since our code works in both 2.7 and 3.4 I think we'll be fairly safe.